wordpress-mobile/WordPress-iOS
# Fix field for category creation.

Fixes #8294 

To test:
 - Start the app
 - Create a new post
 - Tap on the ellipsis button
 - Tap on the Options option
 - Tap on Category
 - Tap on the + 
 - Give a name to the category
 - Tap on save
 - Check that it saves correctly.
